crt_portal/cts_forms/management/commands/update_response_templates.py

```diff
@@ -47,14 +54,26 @@ def handle(self, *args, **options):
                         self.stdout.write(self.style.ERROR(f'Response template {template.name} is missing required `{e.args[0]}` property. Skipping it!'))
                         continue
 
+                    self.template_ids.append(letter_id)
                     # Mark if a letter should be processed from Markdown to HTML.
                     # This is optional. Default value is false
                     # Note: this does not catch errors or typos in values.
                     letter_data['is_html'] = content.get('is_html', False)
+                    letter_data['is_user_created'] = False
 
                     letter, created = ResponseTemplate.objects.update_or_create(title=letter_id, defaults=letter_data)
 
                     if created:
                         self.stdout.write(self.style.SUCCESS(f'Created response template: {letter.title}'))
                     else:
                         self.stdout.write(self.style.SUCCESS(f'Updated response template: {letter.title}'))
+
+        objects_without_templates = filter(self.template_exists, ResponseTemplate.objects.all())
+        for object in objects_without_templates:
+            if not object.is_user_created:
```

Comment:
You can also add this to the filter above to be done at the sql, rather than python, level